### ISSUE TITLE
How do Points get assigned? are they manual?

### ISSUE THREAD

**Issue by llowevad** [OfficeDev/microsoft-teams-apps-champion-management#192]
Is a user supposed to go into the tournament of teams and update their own tasks or is it supposed to be automatic?

If manual, doesn't this leave it open to falsely completing tasks which were not completed?

if automatic, how long is the interval for update?

**Comment by v-saikirang** [comment]
Hi @llowevad,

Thanks for reaching out to us.

The task completion is not automatic. It has to be done by the users manually. There is no way to automatically track the tasks since they tasks can be customized. Additionally you can consider turning on the approvals for tasks by an Admin who can verify and approve. Only after approval the points will get assigned. You can turn the approvals on with an Admin account under "Config Settings" in Admin section.

PLease let me know if you need additional information. 
Thanks.

**Comment by v-saikirang** [comment]
Hi @llowevad,

I am closing this ticket since we don't see any response. Please open a new ticket for any further issues/questions.

Thanks.
